guides/WixApps.md

```diff
@@ -1,20 +1,26 @@
-# About Wix Business Solutions
+# About Apps Made By Wix
+
+Wix sites can contain one or more apps made by Wix. Some API calls require a reference to the specific app made by Wix that it relates to. For example, when calling the [Create Comment](https://dev.wix.com/docs/rest/api-reference/comments/comments/create-comment) API, you must pass an `appId` to specify where to create the comment, for example, in Wix Blogs. Additionally, some API calls return a reference to the app made by Wix. For example, calling the [`Get a Coupon`](https://dev.wix.com/docs/rest/api-reference/coupons/coupons/get-a-coupon) API returns a coupon object with an `appId` referring to the ID of the Wix app that created the coupon, such as Wix Bookings or Wix Events. 
+
+See the following apps made by Wix and their app IDs:
```

Comment:
```suggestion
See the following apps made by Wix and their corresponding IDs. Regardless of the API call, each app always has the same `app ID`.
```

Reply:
Maybe add a note mentioning that other apps that are not made by Wix have their own app IDs, defined by xxx. ... (just as a by-the-way).  

This is just an idea, not a must.